fs/span-manager/span_manager.go

```diff
@@ -303,10 +303,10 @@ func (m *SpanManager) getSpanFromCache(spanId string, offset, size soci.FileSize
 }
 
 func (m *SpanManager) verifySpanContents(compressedData []byte, id soci.SpanId) error {
-	actualDigest := digest.FromBytes(compressedData)
-	expectedDigest := m.ztoc.ZtocInfo.SpanDigests[id]
-	if actualDigest != expectedDigest {
-		return ErrIncorrectSpanDigest
+	actual := digest.FromBytes(compressedData)
```

Comment:
@kzys 
Could you explain why made these changes?  Variable names?  Change in the error?

We'd all really appreciate your thoughts on idiomatic go

Reply:
Previously the error only tells that the digests are different. Including the digests in the error helps troubleshooting the issue. For example, if SpanDigests are somehow not initialized, `expected` could be `""`.

Changing the names is probably more about my preference and I'm fine to revert. The function is short and only handling digests. So `Digest` suffix wouldn't help readers understand the code better.